src/object.d

```diff
@@ -4795,3 +4795,152 @@ TTo[] __ArrayCast(TFrom, TTo)(TFrom[] from) @nogc pure @trusted
     foreach (v; s)
         assert(v == cast(short) 0xabab);
 }
+
+// Yields the appropriate printf format token for a type T
+// Indended to be used by miniFormat
+private template getPrintfFormat(T)
+{
+    static if (is(T == long))
+    {
+        enum getPrintfFormat = "%lld";
+    }
+    else static if (is(T == ulong))
+    {
+        enum getPrintfFormat = "%llu";
+    }
```

Comment:
According to Clang on macOS:

* `c_long`  -> `%ld` 
* `c_ulong` -> `%lu`
* `size_t` on 64bit-> `%lu` 
* `size_t` on 32bit-> `%zu`

Reply:
We don't need to worry about `size_t` as it will be either `ulong` or `uint`, but I changed the formatting for (u)longs to `%lu` and `%ld`.

Reply:
No, it was correct before. `long long` in C is `%lld` and `unsigned long long is `%llu`. It uses different format specifiers for `long` and `long long`. I was thinking of adding additional cases for `c_long`.

Reply:
Regarding `size_t`, it's supposed to use `%zu` on 32bit macOS. Not sure what the difference is compared to `%u`.